src/metrics-modeling/metricbot/extract.py

```
@@ -0,0 +1,112 @@
+        "-h", config.pg_host,
+        "-p", config.pg_port,
+        "-U", config.pg_user,
+        "-d", config.pg_database,
+        "-f", str(sql_file),
+        "--csv",
+        "-v", "ON_ERROR_STOP=1",
+    ]
+    for name, value in (variables or {}).items():
+        args += ["-v", f"{name}={int(value)}"]  # int() — nothing user-shaped reaches psql
+
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)